storage/client/client.go

```diff
@@ -1381,6 +1399,19 @@ func (c *StorageClient) RuntimeEvents(ctx context.Context, p apiTypes.GetRuntime
 		if et != (apiTypes.EvmEventToken{}) {
 			e.EvmToken = &et
 		}
+		// Render Ethereum-compatible address preimages.
+		// TODO: That's a little odd to do in the database layer. Move this farther
+		// out if we have the energy.
+		if fromPreimageContextIdentifier != nil && fromPreimageContextVersion != nil {
+			e.Body["from_eth"] = EthChecksumAddrFromPreimage(*fromPreimageContextIdentifier, *fromPreimageContextVersion, fromPreimageData)
```

Comment:
`body` is now no longer a verbatim copy of the raw event. Reminder to update the openapi docs accordingly.

Reply:
Updated!